modulus/datapipes/cae/domino_datapipe.py

```diff
@@ -233,6 +233,15 @@ def __getitem__(self, idx):
                     volume_coordinates = volume_coordinates[ids_in_bbox]
                     volume_fields = volume_fields[ids_in_bbox]
 
+                # volume_fields[:, :3] = volume_fields[:, :3] / STREAM_VELOCITY
```

Comment:
Do you want to keep it commented out or remove? If the former - maybe add a comment/TODO.